cmd/aries-agent-mobile/README.md

```diff
@@ -77,19 +77,20 @@ import java.nio.charset.StandardCharsets;
 */
         // create options
         Options opts = new Options();
-        opts.setURL("http://example.com");
-        opts.setUseLocalAgent(true);
+        opts.setAgentURL("http://example.com");
+        opts.setUseLocalAgent(false);
 
         ResponseEnvelope res = new ResponseEnvelope();
         try {
             // create an aries agent instance
-            AriesController a = Ariesagent.newAriesAgent(opts);
+            AriesController a = Ariesagent.new_(opts);
```

Comment:
i assume the underscore suffix is a gomobile convention?

Reply:
yes it's autogenerated to make a distinction between the `new` keyword in java